dev/tidb-in-kubernetes/deploy/tidb-operator.md

```diff
@@ -60,6 +60,10 @@ Because TiDB uses many file descriptors by default, the worker node and its Dock
 
     Set `LimitNOFILE` to be greater than or equal to `1048576`.
 
+> **Note:**
```

Comment:
Since this note is for `LimitNOFILE` in step 2, please indent this note with four spaces.